src/components/board/7seg/style.css

```diff
@@ -0,0 +1,171 @@
+.seven-seg-container {
+  height: 4rem;
+  border-radius: 10px;
+  display: flex;
+  width: 33px;
+  margin-right: 5px;
+}
+
+.label {
+  font-size: 0.6rem;
+  text-align: center;
+}
+
+body {
```

Comment:
I would remove this because this now causes the complete page to receive a margin of 48px. So now you have a border around the whole application. I guess this was by accident?

Reply:
I guess, that I will anyways update the styling when integrating this into the user interface branch once it's on master.

Reply:
You are right, I removed that part from the css. It was not really needed.